src/module/actors/sheets/character-sheet.js

```diff
@@ -141,6 +141,11 @@ export class STACharacterSheet extends ActorSheet {
     // This creates a dynamic Stress tracker. It polls for the value of the fitness attribute, security discipline, and checks for Resolute talent. 
     // With the total value, creates a new div for each and places it under a child called "bar-stress-renderer".
     function stressTrackUpdate() {
+
+      var localizedValues = {

```

Comment:
Consider using `const` instead of `var` unless this variable is intended to be modified by other code.